tensorflow/python/ops/image_ops_impl.py

```diff
@@ -45,6 +45,8 @@
 # TODO(b/31222613): This op may be differentiable, and there may be
 # latent bugs here.
 ops.NotDifferentiable('HSVToRGB')
+ops.NotDifferentiable('YIQToRGB')
+ops.NotDifferentiable('RGBToYIQ')
```

Comment:
These look pretty differentiable to me... it's just a linear transformation, no?